parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -219,6 +219,13 @@ time_metric "EnsureContainerd" ensureContainerd
 time_metric "EnsureDHCPv6" ensureDHCPv6
 {{end}}
 
+if [[ -n ${MASTER_NODE} ]]; then
+  {{/* configure and enable kms plugin */}}
+  {{- if EnableEncryptionWithExternalKms}}
```

Comment:
nit: we can save a little cloud-init overhead by moving the `{{- if EnableEncryptionWithExternalKms}}` up the conditional hierarchy (basically we don't need to even bother with this particular `if [[ -n ${MASTER_NODE} ]]; then` part if we don't have a kms config)

Reply:
Updated!